ffi/build.py

```diff
@@ -188,17 +188,21 @@ def main_posix(kind, library_ext):
     else:
         (version, _) = out.split('.', 1)
         version = int(version)
-        if version == 16:
-            msg = ("Building with LLVM 16; note that LLVM 16 support is "
-                   "presently experimental")
+        if version == 20:
+            pass
```

Comment:
This `if .. pass` pattern feels a bit funny. I think we can get rid of it with a change below:
```suggestion
```